lib/activerecord_spanner_adapter/table/column.rb

```diff
@@ -28,6 +31,8 @@ def initialize \
         @ordinal_position = ordinal_position
         @allow_commit_timestamp = allow_commit_timestamp
         @default = default
+        @default_function = default_function
+        @generated = generated != false
```

Comment:
Shouldn't this be `@generated = generated == true`?

Reply:
That will be safer if this class is initialized with `nil`, updated.